### PR TITLE
Export modules under ./stats directory and add type definitions

### PR DESCRIPTION
**Pull request by sarslanhan**
Related to #46

### CODE REVIEW

File: stats/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+exports.EWMA = require('./exponentially_weighted_moving_average');
+exports.ExponentiallyDecayingSample = require('./exponentially_decaying_sample');
+exports.Sample = require('./sample');
+exports.UniformSample = require('./uniform_sample');
```
Comment (tcpshaaban) [inline stats/index.js]: Please leave a new line here

**Comment by tcpshaaban** [comment]
👍

File: stats/index.js (hunk 1 above)
Comment (sarslanhan) [inline stats/index.js]: Done!

File: index.js [hunk 2]
```diff
@@ -14,5 +15,10 @@ exports.ConsoleReporter = Reporting.ConsoleReporter;
 exports.CsvReporter = Reporting.CsvReporter;
 exports.GraphiteReporter = Reporting.GraphiteReporter;
 
-exports.version = '0.1.10';
+exports.EWMA = Stats.EWMA;
+exports.ExponentiallyDecayingSample = Stats.ExponentiallyDecayingSample;
+exports.Sample = Stats.Sample;
+exports.UniformSample = Stats.UniformSample;
+
+exports.version = '0.1.12';
```
Comment (tolbertam) [inline index.js]: I'll find a way to automate this, should be pullable from package.json

**Comment by tolbertam** [comment]
d'oh, I meant to approve, not request changes 🤦‍♂️.  looks great :)

**Pull request merged**